wrappers/rust/icicle-curves/icicle-bn254/src/fri/mod.rs

```diff
@@ -6,9 +6,10 @@ impl_fri!("bn254", bn254_fri, ScalarField, ScalarCfg);
 
 #[cfg(test)]
 mod tests {
-    use icicle_core::impl_fri_tests;
-
-    use crate::curve::ScalarField;
-
-    impl_fri_tests!(bn254_fri_test, ScalarField, ScalarField);
+    mod bn254_fri_test {
```

Comment:
I don't think this extra mod is needed. Its already separated from others by the directory path